src/NUnitTestAdapterInstall/NUnitTestAdapterInstall.csproj

```diff
@@ -30,7 +30,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\bin\Debug\</OutputPath>
+    <OutputPath>bin\Debug\</OutputPath>
```

Comment:
The cake script picks the vsix from this directory. I just wanted to make sure I didn't inadvertently get any wrong dll's due to using a common output directory, because that is what happened.  So, I decided to just have separate output directories for each project, and pick from those.